docs/contributing.rst

```diff
@@ -46,7 +46,7 @@ reStructuredText <http://sphinx.pocoo.org/rest.html>`__.
 The documentation follows the
 `NumPy Docstring Standard <https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>`__,
 which are parsed using the
-`napolean extension for sphinx <http://www.sphinx-doc.org/en/1.5.1/ext/napoleon.html>`.
+`napoleaon extension for sphinx <http://www.sphinx-doc.org/en/1.5.1/ext/napoleon.html>`.
```

Comment:
The `a` sneaked into your fix! Please could you remove it (`napoleaon` -> `napoleon`)?

Reply:
I'm sorry, it has been corrected now.